### PR TITLE
[bn] Localize horizontal-scaling.md

### PR DESCRIPTION
**Pull request by nerdynikhil**
Signed-off-by: Nikhil Barik [nklmarch17@gmail.com](mailto:nklmarch17@gmail.com)
Localizes `content/bn/horizontal-scaling.md` to Bengali
This is an intial PR as may need some improvements!
Tracking issue: #410 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/bn/horizontal-scaling.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: অনুভূমিক পরিমাপ (Horizontal Scaling)
```
Comment (mitul3737) [inline content/bn/horizontal-scaling.md:2]:
```suggestion
title: অনুভূমিক স্কেলিং (Horizontal Scaling)
```

File: content/bn/horizontal-scaling.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+status: Completed
+category: ধারণা
+tags: ["infrastructure", "", ""]
+---
+
+## এটা কি
+
+অনুভূমিক স্কেলিং হল এমন একটি কৌশল যেখানে আরও [নাড](/bn/nodes/) যোগ করে একটি সিস্টেমের ক্ষমতা বৃদ্ধি করা হয়।
```
Comment (mitul3737) [inline content/bn/horizontal-scaling.md:10]:
```suggestion
অনুভূমিক স্কেলিং হল এমন একটি কৌশল যেখানে আরও [নোড](/bn/nodes/) যোগ করে একটি সিস্টেমের ক্ষমতা বৃদ্ধি করা হয়।
```

File: content/bn/horizontal-scaling.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+স্বতন্ত্র নোডগুলিতে আরও গণনা সংস্থান যোগ করার বিপরীতে (পরবর্তীটি [উল্লম্ব পরিমাপে](/bn/vertical-scaling/) নামে পরিচিত)।
```
Comment (mitul3737) [inline content/bn/horizontal-scaling.md:11]:
```suggestion
স্বতন্ত্র নোডগুলিতে আরও গণনা সংস্থান যোগ করার বিপরীতে (পরবর্তীটি [উল্লম্ব স্কেলিং](/bn/vertical-scaling/) নামে পরিচিত)।
```

File: content/bn/horizontal-scaling.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+ধরা যাক, আমাদের 4GB RAM এর একটি সিস্টেম আছে এবং এর ক্ষমতা 16GB RAM-তে বাড়াতে চাই,
+এটিকে অনুভূমিকভাবে স্কেল করার অর্থ হল 16GB RAM সিস্টেমে স্যুইচ করার পরিবর্তে 4 x 4GB RAM যোগ করে তা করা।
+
+এই পদ্ধতিটি নতুন দৃষ্টান্ত, বা [নাড](/bn/nodes/), যোগ করে একটি অ্যাপ্লিকেশনের কর্মক্ষমতা বাড়ায়
```
Comment (mitul3737) [inline content/bn/horizontal-scaling.md:15]:
```suggestion
এই পদ্ধতিটি নতুন দৃষ্টান্ত, বা [নোড](/bn/nodes/), যোগ করে একটি অ্যাপ্লিকেশনের কর্মক্ষমতা বাড়ায়
```

File: content/bn/horizontal-scaling.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+অনুভূমিক পরিমাপ অ্যাপ্লিকেশনগুলিকে অন্তর্নিহিত ক্লাস্টার সরবরাহ করে যা সীমাবদ্ধ করে তা স্কেল করার অনুমতি দেয়।
+সিস্টেমে আরও দৃষ্টান্ত যোগ করে, অ্যাপটি আরও বেশি সংখ্যক অনুরোধ প্রক্রিয়া করতে পারে।
+যদি একটি একক নোড প্রতি সেকেন্ডে 1,000টি অনুরোধ পরিচালনা করতে পারে,
+প্রতিটি অতিরিক্ত নোড প্রতি সেকেন্ডে প্রায় 1,000 অনুরোধ দ্বারা অনুরোধের মোট সংখ্যা বৃদ্ধি করা উচিত।
+এটি অ্যাপ্লিকেশনটিকে একসাথে আরও কাজ করতে দেয়
+বিশেষ করে কোনো নোডের ক্ষমতা বাড়ানোর প্রয়োজন ছাড়াই।
+
+## সম্পর্কিত পদ
+
+* [উল্লম্ব পরিমাপে](/bn/vertical-scaling/)
```
Comment (mitul3737) [inline content/bn/horizontal-scaling.md:38]:
```suggestion
* [উল্লম্ব  স্কেলিং](/bn/vertical-scaling/)
```

**Review by mitul3737 — CHANGES_REQUESTED** [review]

**Review by mitul3737 — APPROVED** [review]

**Review by Imtiaz1234 — APPROVED** [review]

**Comment by asem-hamid** [comment]
Fixes #1866

**Pull request merged**